swarm/storage/chunker.go

```diff
@@ -357,7 +350,9 @@ func (tc *TreeChunker) runWorker() {
 					return
 				}
 
-				h, err := tc.putter.Put(tc.ctx, job.chunk)
+				h, err := tc.putter.Put(ctx, job.chunk)
+				//>>>>>>> master
```

Comment:
???? Leftovers from some merge? Remove comments?

Reply:
Yes, I removed it